package.json

```diff
@@ -11,7 +11,11 @@
     "eslint-plugin-react": "^7.30.1",
     "eslint-plugin-react-hooks": "^4.6.0",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
```

Comment:
Brave of you to try out the carousel package! Guessing this is a great way to learn :)